services/contract_create.proto

```diff
@@ -147,4 +147,14 @@ message ContractCreateTransactionBody {
      * the memo that was submitted as part of the contract (max 100 bytes)
      */
     string memo = 13;
+
+    /**
+     * List with accounts that should be "warmed up"/loaded in the EVM
+     */
+    repeated AccountID account_access = 14;
+
+    /**
+     * List with smart contracts and storage keys that should be "warmed up"/loaded in the EVM
+     */
+    repeated ContractAccessEntry contract_access = 15;
```

Comment:
We should also support these on a contract update and show them on contract info query.

Reply:
The access list is an aspect of a particular transaction, not of a contract as a whole.